### ISSUE TITLE
setclaimflag changebiome error

### ISSUE THREAD

**Issue by andris155** [ShaneBeee/GriefPreventionFlags#88]
### Description

Unhandled exception executing command 'setclaimflag' in plugin GriefPreventionFlags v5.8.3

### Steps to Reproduce

Write /setclaimflag changebiome

### Expected Behavior

Write biomes list?

### Errors / Screenshots

```
[18:56:12] [Server thread/INFO]: Kush1na issued server command: /setclaimflag changebiome
[18:56:12] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'setclaimflag' in plugin GriefPreventionFlags v5.8.3
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[patched_1.15.2.jar:git-Paper-114]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.15.2.jar:git-Paper-114]
	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.dispatchCommand(CraftServer.java:742) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.PlayerConnection.handleCommand(PlayerConnection.java:1821) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:1629) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.PacketPlayInChat.a(PacketPlayInChat.java:47) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.MinecraftServer.ba(MinecraftServer.java:1038) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:1031) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:1102) ~[patched_1.15.2.jar:git-Paper-114]
	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:934) ~[patched_1.15.2.jar:git-Paper-114]
	at java.lang.Thread.run(Unknown Source) [?:?]
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at me.ryanhamshire.GPFlags.GPFlags.onCommand(GPFlags.java:670) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched_1.15.2.jar:git-Paper-114]
	... 17 more
```

### Server Information
* **Server version/platform:** 1.15.2/Paper       
* **GriefPrevention Version:** 16.13.0       
* **GriefPreventionFlags Version:** 5.8.3

**Comment by ShaneBeee** [comment]
I'll take a look into it soon.

**Comment by ShaneBeee** [comment]
Fixed and will appear in next update!

**Comment by andris155** [comment]
Thanks!

**Comment by ShaneBeee** [comment]
Fixed in 5.8.5
